protostuff-runtime/src/main/java/io/protostuff/runtime/RuntimeMapFieldFactory.java

```diff
@@ -46,6 +46,8 @@ private RuntimeMapFieldFactory()
     {
     }
 
+    static final Accessor.Factory AF = ReflectAccessor.FACTORY;
```

Comment:
I'm wondering though if we should rather use `RuntimeFiledFactory.ACCESSOR_FACTORY` here (which will use reflection/unsafe depending on `RuntimeEnv.USE_SUN_MISC_UNSAFE`)?

Reply:
Definitely use `RuntimeFiledFactory.ACCESSOR_FACTORY`.
PR looks good. Thanks!